src/css/components/_volume.scss

```diff
@@ -1,27 +1,32 @@
-.video-js .vjs-mute-control,
-.video-js .vjs-volume-menu-button {
+.video-js .vjs-mute-control {
   cursor: pointer;
   @include flex(none);
   @extend .vjs-icon-volume-high;
 }
 
-.video-js .vjs-mute-control.vjs-vol-0,
-.video-js .vjs-volume-menu-button.vjs-vol-0 {
+.video-js .vjs-mute-control.vjs-vol-0 {
   @extend .vjs-icon-volume-mute;
 }
-.video-js .vjs-mute-control.vjs-vol-1,
-.video-js .vjs-volume-menu-button.vjs-vol-1 {
+.video-js .vjs-mute-control.vjs-vol-1 {
   @extend .vjs-icon-volume-low;
 }
-.video-js .vjs-mute-control.vjs-vol-2,
-.video-js .vjs-volume-menu-button.vjs-vol-2 {
+.video-js .vjs-mute-control.vjs-vol-2 {
   @extend .vjs-icon-volume-mid;
 }
 
 .video-js .vjs-volume-control {
-  width: 5em;
+  width: auto;
+  margin-right: 1em;
+  @include flex(none);
+  @include display-flex(center);
+  @include transition(all 0.4s);
```

Comment:
we should avoid using transition all but instead only transition the specific properties we want. Using `all` here potentially can cause performance issues since the browsers then try to transition all the properties even if we only care about width or height, for example.
We don't have to solve it immediately, but we should do it before a 6.0 release.